scripts/create_gitlab_snapshot.py

```diff
@@ -1,8 +1,15 @@
+"""
+Create a snapshot of the storage volume attached to the GitLab instance.
+Automatically handles stopping and restarting the instance unless --no-restart
+is used.
```

Comment:
No need to duplicate the documentation of specific flags as that creates a brittle coupling.